shifthappens/tasks/ccc.py

```diff
@@ -0,0 +1,74 @@
+"""
+Implementation of CCC: Continuously Changing Corruptions
+Note: This Task only implements the data reading portion of the dataset.
+In addition to this file, we submitted a file used to generate the data itself.
+"""
```

Comment:
```suggestion
"""CCC: Continuously Changing Corruptions

.. note::

    This task only implements the data reading portion of the dataset.
    In addition to this file, we submitted a file used to generate the
    data itself.
"""
```

Ideally also add a short description of the dataset here.